plugins/woocommerce-admin/client/payments/payment-settings-banner.tsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * External dependencies
+ */
+import { Card, CardFooter, CardBody, Button } from '@wordpress/components';
+import { Text } from '@woocommerce/experimental';
+import { __ } from '@wordpress/i18n';
+import {
+	ONBOARDING_STORE_NAME,
+	PAYMENT_GATEWAYS_STORE_NAME,
+	Plugin,
+	PaymentGateway,
+	WCDataSelector,
+} from '@woocommerce/data';
+import { useSelect } from '@wordpress/data';
+import { useExperiment } from '@woocommerce/explat';
+
+/**
+ * Internal dependencies
+ */
+import {
+	Visa,
+	MasterCard,
+	Amex,
+	ApplePay,
+	Giropay,
+	GooglePay,
+	CB,
+	DinersClub,
+	Discover,
+	UnionPay,
+	JCB,
+	Sofort,
+} from '../payments-welcome/cards';
+import WCPayBannerImage from './wcpay-banner-image';
+import './payment-recommendations.scss';
+
+export const PaymentMethodsIcons = () => (
+	<div className="woocommerce-recommended-payments-banner__footer_icon_container">
+		<Visa />
+		<MasterCard />
+		<Amex />
+		<ApplePay />
+		<GooglePay />
+		<CB />
+		<Giropay />
+		<DinersClub />
+		<Discover />
+		<UnionPay />
+		<JCB />
+		<Sofort />
+	</div>
+);
+
+const WcPayBanner = () => {
+	const WC_PAY_SETUP_URL =
+		'./admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments';
```

Comment:
I think we could use `getAdminLink` here:

https://github.com/woocommerce/woocommerce/blob/feba6a8dcd55d4f5c7edc05478369c76df082293/plugins/woocommerce-admin/client/tasks/fills/products/products.js#L64-L66

Reply:
Hmm... that's new to me... however I can't actually find the source for @woocommerce/settings and the only mention I've seen of it is in the changelog where it says its been removed

How does this work? 👀👀👀

Reply:
Hmm...I think it's still injected into our code via the Dependency Extraction Webpack Plugin, though I also don't know how it exactly works.

Reply:
>however I can't actually find the source for @woocommerce/settings and the only mention I've seen of it is in the changelog where it says its been removed

The settings package used to be under WCAdmin, but now lives in blocks https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/trunk/assets/js/settings/shared

P2: p90Yrv-2B5-p2

Reply:
Changed in 3ee2fcaeac2a62e92a40ebe0c17f37b90d0fe422